### PR TITLE
Update release_id to fixed_version_id for removal of Backlogs

### PR DESCRIPTION
**Pull request by GregSutcliffe**
~Don't merge yet.~ Good to go.

This is the core changes to the code to start the review process. I'm going through the configs to update the release_ids to match the versions in Redmine, will update the PR once they're ready. I've done one Foreman and one Katello file for testing.

@ehelms could you check this works as expected?

### CODE REVIEW

**Comment by GregSutcliffe** [comment]
Added a second commit with the config file changes in, feel free to squash on merge.

**Comment by jturel** [comment]
I think you can also take [WIP] off of the PR name :)

File: lib/tool_belt/redmine/project.rb [hunk 1]
```diff
@@ -28,7 +28,7 @@ def get_issues(params = {})
     end
 
     def get_issues_for_release(id)
```
Comment (ekohl) [inline lib/tool_belt/redmine/project.rb]: This now matches the `get_issues_for_version` method. We can probably just remove this and switch all callers. We should also get rid of the release naming in favor of version where it's Redmine terms.

Comment (ehelms) [reply lib/tool_belt/redmine/project.rb]: Let's do as a follow up issue as I want to make sure use of these configs is accurate.

**Pull request merged**